main.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_vpc" "main" {
+  cidr_block       = var.cidr
+  instance_tenancy = var.instance_tenancy
+
+  tags = var.vpc_tags
+}
+
+resource "aws_subnet" "main" {
+  vpc_id     = aws_vpc.main.id
+  cidr_block = "10.0.1.0/24"
```

Comment:
move hardcoded value to var.cidr_subnet
create a variable in variables.tf with a default value "10.0.1.0/24"